cthulhu/cthulhu/manager/server_monitor.py

```diff
@@ -324,7 +324,9 @@ def on_mon_map(self, mon_map):
             services = {mon['name']: {'fsid':
                                       mon_map['fsid'],
                                       'type': 'mon',
-                                      'status': '{}',
+                                      'status': {'election_epoch': mon_map['epoch'],
```

Comment:
election epoch and the mon map epoch are two different things, it doesn't make any sense to assign one to the other